embeds.js

```diff
@@ -266,7 +266,45 @@ const embeds = {
     
     devs_only: new EmbedBuilder()
     .setDescription("Only developers of <@997670790604542012> can use this.")
-    .setColor("ED4245")
+    .setColor("ED4245"),
+    selfhostserver: new EmbedBuilder() //If you want to make more complex embeds like this I recommend https://embed.dan.onl/ a-m-a-z-i-n-g
```

Comment:
try to shorten this please, like how the other command is written

Reply:
Is this commit short enough?

Reply:
ye looks good!